app/forms/PersonalDetailsForm.scala

```diff
@@ -137,19 +152,26 @@ object GeneralDetailsForm {
     }
 
     def toExchange(email: String, updateApplicationStatus: Option[Boolean],
-                   overrideCivilServiceExperienceDetails: Option[CivilServiceExperienceDetails] = None) = GeneralDetails(
-      firstName,
-      lastName,
-      preferredName,
-      email,
-      LocalDate.parse(s"${dateOfBirth.year}-${dateOfBirth.month}-${dateOfBirth.day}"),
-      outsideUk.getOrElse(false),
-      address,
-      postCode.map(p => PostCodeMapping.formatPostcode(p)),
-      country,
-      phone,
-      overrideCivilServiceExperienceDetails.orElse(civilServiceExperienceDetails),
-      updateApplicationStatus
-    )
+                   overrideCivilServiceExperienceDetails: Option[CivilServiceExperienceDetails] = None) = {
+      PersonalDetails(
+        firstName,
+        lastName,
+        preferredName,
+        email,
+        LocalDate.parse(s"${dateOfBirth.year}-${dateOfBirth.month}-${dateOfBirth.day}"),
+        outsideUk.getOrElse(false),
+        address,
+        postCode.map(p => PostCodeMapping.formatPostcode(p)),
+        country,
+        phone,
+        overrideCivilServiceExperienceDetails.orElse(civilServiceExperienceDetails),
+        edipCompleted match {
```

Comment:
Unneccesary pattern matching may be use some thing like 
edipCompleted.map(_toBoolean)